src/fsd/4-entities/equipment/equipment.service.ts

```
@@ -0,0 +1,121 @@
+                icon = EquipmentType[slot as keyof typeof EquipmentType] + '_Item';
+                break;
+            case EquipmentType.BlockBooster:
+                icon = 'Block_Booster';
+                break;
+            case EquipmentType.CritBooster:
+                icon = 'Crit_Booster';
+                break;
+        }
+        return 'equipment/' + icon + '_Icon.webp';
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)